### PR TITLE
issues resloved

### PR DESCRIPTION
**Pull request by srikanthgudipudi**

### CODE REVIEW

File: src/app/project-detail/project-detail.page.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
           {{project.status | camelToTitle}}
         </ion-col>
       </ion-row>
-      <div class="custom-filed" *ngIf="project?.learningResources">
+      <div class="custom-filed" *ngIf="project?.learningResources.length">
```
Comment (kiranharidas187) [inline src/app/project-detail/project-detail.page.html]: project?.learningResources?.length

Add like this

File: src/app/profile-update/profile-update.page.ts [hunk 2]
```diff
@@ -58,12 +58,32 @@ export class ProfileUpdatePage implements OnInit {
...
 
   getProfileData() {
+    let state;
     this.toast.startLoader();
     const config = {
       url: `${urlConstants.API_URLS.GET_PROFILE}`
     }
     this.showLoader = true;
     this.kendraService.get(config).subscribe(data => {
+      if (data.result.roles && data.result.roles.length) {
```
Comment (kiranharidas187) [inline src/app/profile-update/profile-update.page.ts:68]: Is this completely working? I f not please remove this change

Comment (srikanthgudipudi) [reply src/app/profile-update/profile-update.page.ts:68]: working, i checked that issue is resolved from backend as well

File: src/app/project-detail/project-detail.page.html (hunk 1 above)
Comment (srikanthgudipudi) [reply src/app/project-detail/project-detail.page.html]: modified

**Pull request merged**